src/components/scenes/existingAccout/ChangeRecoveryScene.tsx

```diff
@@ -165,21 +164,14 @@ export const ChangeRecoveryScene = ({ branding }: Props) => {
     })
   }
   const renderSaveRecoveryMessage = (showEmailPrompt: boolean) => {
-    const paddingRem = [0, 0.5]
     return (
-      <View style={styles.confirmRecoveryMessageContainer}>
-        <ModalMessage paddingRem={paddingRem}>
-          {s.strings.recovery_save_hint_token}
-        </ModalMessage>
-        <ModalMessage isWarning paddingRem={paddingRem}>
-          {s.strings.recovery_save_hint_username_answers}
-        </ModalMessage>
-        {showEmailPrompt ? (
-          <ModalMessage paddingRem={paddingRem}>
-            {s.strings.recovery_save_email_prompt}
-          </ModalMessage>
-        ) : null}
-      </View>
+      <Text style={styles.messageText}>
```

Comment:
Instead of using `<Text>` with a custom CSS, can you keep using the `ModalMessage` component, but put the styles warning text inside?

```ts
<ModalMessage>
  {part1}
  <Text style={styles.warningText}>{part2}</Text>
  {part3}
</ModalMessage>
```

Reply:
Changed to `ModalMessage`.